src/stats/histogram.jl

```diff
@@ -0,0 +1,76 @@
+const histogram_plot_types = [BarPlot, Heatmap, Volume]
+
+function convert_arguments(P::Type{<:AbstractPlot}, h::StatsBase.Histogram{<:Any, N}) where N
+    ptype = plottype(P, histogram_plot_types[N])
+    f(edges) = edges[1:end-1] .+ diff(edges)./2
+    kwargs = N == 1 ? (; width = step(h.edges[1])) : NamedTuple()
+    to_plotspec(ptype, convert_arguments(ptype, map(f, h.edges)..., Float64.(h.weights)); kwargs...)
+end
+
+"""
+    histogram(values; bins = 15, normalization = :none)
+
+Plot a histogram of `values`. `bins` can be an `Int` to create that
+number of equal-width bins over the range of `values`.
+Alternatively, it can be a sorted iterable of bin edges. The histogram
+can be normalized by setting `normalization`. Possible values are:
+
+*  `:pdf`: Normalize by sum of weights and bin sizes. Resulting histogram
+   has norm 1 and represents a PDF.
+* `:density`: Normalize by bin sizes only. Resulting histogram represents
+   count density of input and does not have norm 1. Will not modify the
+   histogram if it already represents a density (`h.isdensity == 1`).
+* `:probability`: Normalize by sum of weights only. Resulting histogram
+   represents the fraction of probability mass for each bin and does not have
+   norm 1.
+*  `:none`: Do not normalize.
+
+## Attributes
+$(ATTRIBUTES)
+"""
+@recipe(Histogram, values) do scene
+    Attributes(
+        bins = 15, # Int or iterable of edges
+        normalization = :none
```

Comment:
I've changed it from `relative = false` as it was before, because I've realized that StatsMakie and AbstractPlotting were defaulting to two different normalizations (StatsMakie was doing `pdf`, whereas here it was `probability`). I guess it's safer to be explicit because different people may have different expectations.